opm/material/fluidsystems/blackoilpvt/Co2GasPvt.hpp

```diff
@@ -66,29 +70,46 @@ class Co2GasPvt
 
     explicit Co2GasPvt() = default;
 
-    explicit Co2GasPvt(const std::vector<Scalar>& salinity,
+    explicit Co2GasPvt(const ContainerT& salinity,
```

Comment:
Remove explicit from constructors that do not take exactly one argument, including the default one above!